app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/EventLogItemViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.app.devoptions.vieweventlogs
+
+import org.oppia.android.app.model.EventLog
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import org.oppia.android.util.system.OppiaDateTimeFormatter
+import javax.inject.Inject
+
+/** [ViewModel] for displaying a event log. */
+class EventLogItemViewModel @Inject constructor(
+  val eventLog: EventLog,
+  private val oppiaDateTimeFormatter: OppiaDateTimeFormatter
+) : ObservableViewModel() {
+
+  fun processDateAndTime(): String {
```

Comment:
All non-private members of a class should have KDocs since they're part of that class's exported API. The KDocs should explain what the function does to provide more context to the caller on how to properly use it.

Reply:
KDoc added. PTAL.